### PR TITLE
Mark share private link header for translation

### PR DESCRIPTION
**Pull request by laurentS**
## Relevant issue(s)
None found

## What does this do?
Marks a string for translation

## Why was this needed?
Foreign languages are great :)

### CODE REVIEW

**Review by garethrees — APPROVED** [review]
Oops, thanks!

File: app/views/request/_share_by_link.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <% if can? :share, info_request %>
   <div class="sidebar__section share-private-url">
-    <h2>Share with private link</h2>
+    <h2>_('Share with private link')</h2>
```
Comment (gbp) [inline app/views/request/_share_by_link.html.erb:3]: Will need to be:

```suggestion
    <h2><%= _('Share with private link') %></h2>
```

Comment (garethrees) [reply app/views/request/_share_by_link.html.erb:3]: oops, shouldn't code review while on a call 🤦

**Comment by gbp** [comment]
Merged 201f38f05f11233f3b10dba6b270075c7e9afa18

**Pull request closed without merging**